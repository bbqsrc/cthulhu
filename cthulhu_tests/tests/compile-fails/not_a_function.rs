#[cthulhu::invoke]
struct Foo {}

#[cthulhu::invoke]
enum Foo {}

#[cthulhu::invoke]
union Foo {}

fn main() {}
