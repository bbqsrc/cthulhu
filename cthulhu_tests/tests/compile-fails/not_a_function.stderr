error: failed to invoke cthulhu: only supported on functions right now
 --> $DIR/not_a_function.rs:2:1
  |
2 | struct Foo {}
  | ^^^^^^

error: failed to invoke cthulhu: only supported on functions right now
 --> $DIR/not_a_function.rs:5:1
  |
5 | enum Foo {}
  | ^^^^

error: failed to invoke cthulhu: only supported on functions right now
 --> $DIR/not_a_function.rs:8:1
  |
8 | union Foo {}
  | ^^^^^
